test/Microsoft.AspNet.Mvc.TagHelpers.Test/InputTagHelperTest.cs

```diff
@@ -820,9 +823,9 @@ public static TheoryData MultiAttributeCheckBoxData
 
             Assert.True(output.SelfClosing);
             Assert.Equal(expectedAttributes, output.Attributes);
-            Assert.Empty(output.PreContent);
-            Assert.Equal(new[] { string.Empty }, output.Content);
-            Assert.Empty(output.PostContent);
+            Assert.Empty(output.PreContent.ToString());
+            Assert.Equal(string.Empty, output.Content.ToString());
+            Assert.Empty(output.PostContent.ToString());
```

Comment:
Also now checking something different than before. An empty `TagHelperContent` instance is similar to one that contains only a single `string.Empty` instance. But this method was confirming the _exact_ behaviour e.g. that the tag helper did not interact with `PreContent` at all. (Checking `Assert.False(output.PreContent.IsModified)` is closer to the existing code than this. But will be able to restore the original code once `TagHelperContent` again implements `IEnumerable<string>`.)